src/audio/rtnr/rtnr.c

```diff
@@ -741,22 +752,25 @@ static void rtnr_copy_to_sof_stream(struct audio_stream *dst,
 }
 
 /* copy and process stream data from source to sink buffers */
-static int rtnr_copy(struct comp_dev *dev)
```

Comment:
@MingJenTai The code looks that there could be multiple source buffers and one sink. But it also looks from consume than there would be now one supported. Can you confirm the number of sources to support, one or many?  Thanks!

Reply:
The topology https://github.com/thesofproject/sof/blob/main/tools/topology/topology1/sof/pipe-rtnr-google-rtc-audio-processing-capture.m4 used one source and sink so I will test this work with similar number of them.